worker-default-configs/README.md

```diff
@@ -39,13 +39,14 @@ The default Rabbit configuration file checks for values as below;
 
 The default RabbitWorkerQueue configuration file checks for values as below;
 
-| Property | Checked Environment Variables | Default               |
-|----------|-------------------------------|-----------------------|
-| prefetchBuffer  |  `CAF_RABBITMQ_PREFETCH_BUFFER` | 1  |
-| inputQueue  |  `CAF_WORKER_INPUT_QUEUE` | worker-in  |
-|             |  `CAF_WORKER_BASE_QUEUE_NAME` with '-in' appended to the value if present    |    |
-|             |  `CAF_WORKER_NAME` with '-in' appended to the value if present        |    |
-| pausedQueue  |  `CAF_WORKER_PAUSED_QUEUE` |   |
-| retryQueue  |  `CAF_WORKER_RETRY_QUEUE` |   |
-| rejectedQueue  |   | worker-rejected  |
-| retryLimit  |  `CAF_WORKER_RETRY_LIMIT` | 10  |
+| Property | Checked Environment Variables                                                         | Default        |
+|----------|---------------------------------------------------------------------------------------|----------------|
+| prefetchBuffer  | `CAF_RABBITMQ_PREFETCH_BUFFER`                                                        | 1              |
+| inputQueue  | `CAF_WORKER_INPUT_QUEUE`                                                              | worker-in      |
+|             | `CAF_WORKER_BASE_QUEUE_NAME` with '-in' appended to the value if present              |                |
+|             | `CAF_WORKER_NAME` with '-in' appended to the value if present                         |                |
+| pausedQueue  | `CAF_WORKER_PAUSED_QUEUE`                                                             |                |
+| retryQueue  | `CAF_WORKER_RETRY_QUEUE`                                                              |                |
+| rejectedQueue  |                                                                                       | worker-rejected |
+| retryLimit  | `CAF_WORKER_RETRY_LIMIT`                                                              | 10             |
+|   | `CAF_WORKER_FRIENDLY_NAME` Should be set to preferred worker name rather than default | worker.getClass().getName()         |
```

Comment:
Unfortunately CAF_WORKER_FRIENDLY_NAME isn't related to the RabbitWorkerQueue configuration file.

I see that CAF_WORKER_RETRY_LIMIT is documented in Error-Handling.md so let's update that to explain how CAF_WORKER_FRIENDLY_NAME is used.